packages/eas-cli/src/utils/download.ts

```diff
@@ -94,34 +92,51 @@ async function downloadFileWithProgressBarAsync(
   await pipeline(response.body, fs.createWriteStream(outputPath));
 }
 
-export async function downloadAndExtractAppAsync(
+export async function downloadAndMaybeExtractAppAsync(
   url: string,
-  applicationExtension: string
+  platform: AppPlatform
 ): Promise<string> {
   const outputDir = path.join(getTmpDirectory(), uuidv4());
   await fs.promises.mkdir(outputDir, { recursive: true });
 
-  const tmpArchivePathDir = path.join(getTmpDirectory(), uuidv4());
-  await fs.promises.mkdir(tmpArchivePathDir, { recursive: true });
-
-  const tmpArchivePath = path.join(tmpArchivePathDir, `${uuidv4()}.tar.gz`);
-
-  await downloadFileWithProgressBarAsync(url, tmpArchivePath, 'Downloading app archive...');
-  await tarExtractAsync(tmpArchivePath, outputDir);
+  if (platform === AppPlatform.Android) {
+    const apkFilePath = path.join(outputDir, `${uuidv4()}.apk`);
```

Comment:
Is it always an .apk?

Reply:
I've always seen `.apk` for my apps built from the example project, although you are right I will make it more elastic and support both `apk` and tarball for Android 👍 .